### ISSUE TITLE
xCode 7.0 dosent get found by robovm plugin

### ISSUE THREAD

**Issue by sgedwardlim** [robovm/robovm#1161]
<img width="974" alt="screen shot 2015-09-22 at 12 57 13 pm" src="https://cloud.githubusercontent.com/assets/9358157/10029901/80a42f4e-6129-11e5-9286-550a9e547239.png">

**Comment by badlogic** [comment]
Where did you install Xcode to? /Applications/Xcode.app?

**Comment by badlogic** [comment]
Couls you also try `xcode-select -p` on the command line and tell me what the output is?

**Comment by sgedwardlim** [comment]
This is the output I get when I run xcode-select -p

| => xcode-select -p
-bash: xcode-select: command not found

<img width="769" alt="screen shot 2015-09-22 at 5 11 33 pm" src="https://cloud.githubusercontent.com/assets/9358157/10035257/03739e82-614d-11e5-8c1b-49a257050aa1.png">

And as to where I installed xcode to, the picture should show the location, I belieive thats the answer you were looking for hopefully

**Comment by sgedwardlim** [comment]
Just to add on to what I did this morning prior to this issue. I received an email from apple stating that there were some compromised versions of xcode that were infected with the XcodeGhost and after doing some of the recommended test I found out that my xcode (preinstalled) was infected so I proceeded to uninstall it and re-downloaded it now from the app store, and I verified that it was the right one. Not sure if this helps but here it is. Also here is what happens when I key echo $PATH

/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/git/bin
